AR-builder.groovy

```diff
@@ -527,9 +559,12 @@ pipeline {
 
                     postGithubComment(params.GITHUB_CHANGE_ID, payload)
                 }
+
+                if (env.SOURCE_BRANCH ==~ /v[0-9]+.[0-9]+.[0-9A-Za-z-+]+/) {
+                    postGithubApkToRelease(params.FLAVOR, params.BUILD_TYPE)
+                }
             }
 
-            sh './gradlew jacocoReport'
```

Comment:
This is not needed at all in Jenkins. Confirmed with QA.
We only run coverage in GH actions.